src/main/java/su/plo/voice/listeners/PlayerListener.java

```diff
@@ -75,6 +75,15 @@ public void onPlayerJoin(PlayerJoinEvent e) {
                     String.format("Voice chat is installed but doesn't work. Check if port %d UDP is open.",
                     PlasmoVoice.getInstance().config.port));
         }
+
+        Bukkit.getScheduler().runTaskLater(PlasmoVoice.getInstance(), () -> {
```

Comment:
Also, It's better to first check if the config has "clientModRequired" set as true, and then do runTaskLater

Reply:
Done